compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -218,21 +217,24 @@ function finishCallStmt(Tokenizer tok, CallStmt expr, Position startPos) returns
     return stmt;
 }
 
+// call after ingesting checkingKeyword startPos set to position of checkingKeyword
```

Comment:
This comment is not grammatical English. Please start comments that are on a line by themselves with a capital letter,

Reply:
fixed